flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/operators/window/tvf/slicing/SliceAssigners.java

```diff
@@ -566,6 +592,11 @@ public long getSliceEndInterval() {
             return innerAssigner.getSliceEndInterval();
         }
 
+        @Override
+        public long getWindowOffset() {
+            return 0;
```

Comment:
If this class is abstract then you don't need to return anything.

Reply:
I am curious, if the children do not override this why is the value 0, where everywhere else in the code this getter returns offset.

Reply:
> If this class is abstract then you don't need to return anything.

Thank you for your reminder. I have change it to "return innerAssigner.getWindowOffset()"